solver/src/solver/http_solver.rs

```diff
@@ -293,10 +293,12 @@ impl HttpSolver {
         let now = chrono::Utc::now();
         format!(
             "{}_{}_{}",
-            now.to_string().replace(" ", "_"),
+            now.to_string(),
```

Comment:
For the time it is better to use explicit format arguments https://docs.rs/chrono/0.4.19/chrono/struct.DateTime.html#method.format instead of using the default to_string and then fixing characters.